packages/electron/lib/install.js

```diff
@@ -94,42 +118,59 @@ module.exports = {
     })
   },
 
+  async getRealArch (platform, arch) {
+    if (platform === 'darwin' && arch === 'x64') {
+      // see this comment for explanation of x64 -> arm64 translation
+      // https://github.com/cypress-io/cypress/pull/25014/files#diff-85c4db7620ed2731baf5669a9c9993e61e620693a008199ca7c584e621b6a1fdR11
+      return systeminformation.cpu()
```

Comment:
@flotwig using the updated `systeminformation` npm package here instead of `systemctl.proc_translated` which is what we do in the CLI module

Reply:
nice, I didn't know systeminformation could do this.